freediscovery/interpretation.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+import matplotlib as mpl
+import matplotlib.cm as cm
+import numpy as np
+
+
+def explain_logistic_regression(weights, textlines):

```

Comment:
let's call this `explain_categorization` as while it would indeed work first for LogisticRegression, nothing in this function is exclusive to Logisitic Regression.

Also it should take in a `text` which is a string, not a list of strings, we could apply `.splitlines()` method to it if the later is needed inside the function. Should also take in the `colormap` parameter (as all the underlying functions).